src/cmd/scan.rs

```diff
@@ -52,6 +53,7 @@ pub fn extract_data(
     conf: &config::Config,
     git_repo: &repo::Repo,
 ) -> Result<extractor::Data, String> {
+    let mut spin = Spinner::new(Spinners::Dots, "Extracting data...".to_string());
```

Comment:
`Spinners::Dots` characters seems to not work (at least natively) on some terminals (for example Ubuntu on Windows and Windows PowerShell) while `Spinners::Line` characters work.
It seems that `Spinners::Line` characters work more consistently across multiple terminals.
There are also other Spinners, but I have not tested them.